src/components/Spinner/Spinner.stories.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { withKnobs, select, boolean } from '@storybook/addon-knobs';
+
+import { defaultTemplate } from '../../../storybook/decorators/storyTemplates';
+import Spinner from './index';
+
+const stories = storiesOf('Widgets', module);
+
+stories.addDecorator(withKnobs);
+stories.addDecorator(
+  defaultTemplate({
+    title: 'Spinner',
+    documentationLink:
+      'http://www.patternfly.org/pattern-library/widgets/#spinner',
+  }),
+);
+
+stories.addWithInfo('Spinner', () => {
+  const loading = boolean('Loading', true);
+  const inline = boolean('Inline', false);
+  const inverse = boolean('Inverse', false);
+  const size = select('Size', ['lg', 'md', 'sm', 'xs'], 'md');
+
+  const wrapperStyle = {
+    backgroundColor: inverse ? 'black' : 'white',
+    color: inverse ? 'white' : 'black',
+    padding: '15px',
+  };
+
+  const spinnerProps = {
+    loading,
+    size,
+    inline,
+    inverse,
+  };
+
+  if (inline) {
+    return (
+      <div style={wrapperStyle}>
+        <Spinner {...spinnerProps} />
+        Some inline text here.
+      </div>
+    );
+  }
+
+  return (
+    <div style={wrapperStyle}>
+      <Spinner {...spinnerProps}>
+        <strong>
```

Comment:
I haven't seen this use case in the current test pages, but maybe I am missing something.

@jgiardino thoughts?

Reply:
@priley86 Can you elaborate on the use case you're referring to? I'm guessing you're referring to the use of `<strong>` here. But based on what I'm seeing in the storybook, the contents inside the `<Spinner>` component are the contents that display when loading is false, so the use of `<strong>` here is really just a placeholder for those contents, and not really part of the design pattern.

Reply:
@jgiardino yeah that's correct.